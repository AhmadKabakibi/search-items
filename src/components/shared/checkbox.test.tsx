import 'jest-styled-components'
import React from 'react'

import Checkbox from './checkbox'

describe('Checkbox', () => {
  const Component = shallow(
    <Checkbox name={'test'} onChange={jest.fn()}>
      {'checkbox label'}
    </Checkbox>
  )
  it('should render a Checkbox', () => {
    expect(Component).toMatchSnapshot()
  })
})
