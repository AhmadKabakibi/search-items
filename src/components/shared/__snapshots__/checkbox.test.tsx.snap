// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox should render a Checkbox 1`] = `
<checkbox__Container>
  <checkbox__Input
    name="test"
    onChange={[MockFunction]}
    type="checkbox"
  />
  <checkbox__Marker />
  <checkbox__Label>
    checkbox label
  </checkbox__Label>
</checkbox__Container>
`;
